zerver/webhooks/mention/doc.md

```diff
@@ -1,27 +1,38 @@
+# Zulip Mention integration
+
 Get Mention notifications within Zulip via Zapier!
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
 1. Go to your Mention feed, and click on your profile in the top-right
    corner. Select **Settings**. Click on **Integrations**. Select the
    **Zapier** integration.
 
 1. Click **Explore Mention on Zapier**. Search for "webhooks" in
-   the search bar, and click on **Webhooks by Zapier**. Scroll down
-   and look for **Add Webhook posts for new Mentions**, and click on
+   the search bar, and click on **Webhooks by Zapier**. Scroll down,
```

Comment:
I think we're supposed to remove phrases like "Scroll down" as per the guidelines.